docs/guides/high-availability/concepts-multi-site.adoc

```diff
@@ -100,17 +97,22 @@ The statement "`No data loss`" depends on the setup not being degraded from prev
 == Known limitations
 
 Site Failure::
-* A successful failover requires a setup not degraded from previous failures.
+A successful failover requires a setup not degraded from previous failures.
 All manual operations like a re-synchronization after a previous failure must be complete to prevent data loss.
 Use monitoring to ensure degradations are detected and handled in a timely manner.
 
 Out-of-sync sites::
-* The sites can become out of sync when a synchronous {jdgserver_name} request fails.
+The sites can become out of sync when a synchronous {jdgserver_name} request fails.
 This situation is currently difficult to monitor, and it would need a full manual re-sync of {jdgserver_name} to recover.
 Monitoring the number of cache entries in both sites and the {project_name} log file can show when resynch would become necessary.
 
 Manual operations::
-* Manual operations that re-synchronize the {jdgserver_name} state between the sites will issue a full state transfer which will put a stress on the system (network, CPU, Java heap in {jdgserver_name} and {project_name}).
+Manual operations that re-synchronize the {jdgserver_name} state between the sites will issue a full state transfer which will put a stress on the system (network, CPU, Java heap in {jdgserver_name} and {project_name}).
```

Comment:
Suggesting to remove a redundant sentence. Also, want to add that Keycloak should not receive any extra load when the state transfer happens in the Infinispan cluster. 
```suggestion
Manual operations that re-synchronize the {jdgserver_name} state between the sites will issue a full state transfer which will put a stress on the system.
```

Reply:
What do you mean the system should not receive any extra load? I thought in the situation where only one site is active and we are pushing state from active -> recovered the active site can still receive load. If this is not true, should we update our https://github.com/keycloak/keycloak/blob/main/docs/guides/high-availability/operate-synchronize.adoc guide to instruct people to stop all load on Keycloak?